clusterloader2/testing/density/rc.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: {{.Basename}}
+  name: {{.Basename}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    name: {{.Basename}}
+  template:
+    metadata:
+      labels:
+        name: {{.Basename}}
+        phase: phase1
```

Comment:
Can we use a better label (for e.g `app: pause`)? As I think this is not really tied to phases.